README.md

```diff
@@ -52,7 +53,29 @@ into a folder on your host system and then mount this directory into your `docke
 
 Get these packages either from the raspberry, or download them online.
 
-If you do `apt-cache show libssl1.0.0` on the raspberry, you'll see this in the
+#### Using apt-get to fetch packages
+From a debian-based system, you may use apt-get in order to fetch packages directly from the raspbian archives. A script is provided for these 
+purposes in the `apt/` folder.
+
+Here is how you would download libssl1.0 and all of its dependencies. 
+```
+$ cd apt/
+# Install the raspbian keys into your local apt keychain
+$ sudo ./install-keys.sh
```

Comment:
Is this file missing in the commit? (or am I missing something?)

Reply:
indeed